diff --git a/src/PdfParser/StreamReader.php b/src/PdfParser/StreamReader.php
index 27de6d9..5e172a8 100644
--- a/src/PdfParser/StreamReader.php
+++ b/src/PdfParser/StreamReader.php
@@ -119,14 +119,6 @@ class StreamReader
         $this->reset();
     }
 
-    /**
-     * The destructor.
-     */
-    public function __destruct()
-    {
-        $this->cleanUp();
-    }
-
     /**
      * Closes the file handle.
      */
